src/external-types/markdown-it-plantuml/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'markdown-it-plantuml' {
+  import MarkdownIt from 'markdown-it/lib'
+  const markdownItPlantuml: MarkdownIt.PluginWithOptions
```

Comment:
It looks as if the options are missing here. Compare with this other markdown-it-plugin for example: [markdown-it-toc-done-right](https://github.com/codimd/react-client/blob/master/src/external-types/markdown-it-toc-done-right/index.d.ts#L3-L4)

Reply:
done